Project.toml

```diff
@@ -10,6 +10,7 @@ EllipsisNotation = "1.0"
 FillArrays = "0.13.2"
 ForwardDiff = "0.10.18"
 GeometryBasics = "0.3, 0.4"
+GridVisualize = "0.5"
```

Comment:
It seems to me like this feature adds a lot of plotting related dependencies (each of which have their own dependencies). I wonder if this is getting out of hand somewhat, but I don't have a good solution to it either. Maybe we should start thinking about a "TrixiVisualize.jl" or "TrixiPlots.jl" package?
cc @ranocha @jlchan

Reply:
Sounds like a good idea. We should probably discuss this further also at one of the next meetings.